tests/scripts/testcases_run.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 set -euxo pipefail
 
-export PYPATH=$([[ ! "$CI_JOB_NAME" =~ "coreos" ]] && echo /usr/bin/python || echo /opt/bin/python)
```

Comment:
In Ansible 2.8 there is the Python interpreter discovery, so I suppose that's why you removed the `PYPATH` and `ansible_python_interpreter`, but we still need something for coreos & flatcar where Python is in `/opt/bin/python`?

Reply:
can you start the coreos job  and we will see if it works ?
I don't see coreos in https://github.com/ansible/ansible/blob/devel/lib/ansible/config/base.yml, but if python is in the path it will work.

Reply:
tf-ovh_coreos-calico worked actually

Reply:
That's a good cleanup ^^